Proposed_First_FLEDGE_OT_Details.md

```diff
@@ -67,6 +67,16 @@ The FOT#1 will include event-level reporting for both winning and losing bids. I
 
 If the bid being generated or scored loses the auction, the URL will be fetched. These worklets may also call a `forDebuggingOnly.reportAdAuctionWin()` API which operates similarly to `forDebuggingOnly.reportAdAuctionLoss()` API but only fetches the URL after a winning bid or score.
 
+If a seller rejected a bid from participating in an auction, the seller can optionally add a rejectReason field to its return object. The buyer can learn about the reject reason through calling `forDebuggingOnly.reportAdAuctionLoss()` API in  `generateBid()`, by using “${rejectReason}” in the report URL's query string. The reject reason returned by `scoreAd()` should be one of
+*   "not-available"
+*   "invalid-bid"
+*   "bid-below-auction-floor"
+*   "pending-or-disapproved-by-exchange"
```

Comment:
It might be useful to distinguish between the cases of "pending approval" and "disapproved" by a seller.